libs/manubot_ai_editor/models.py

```diff
@@ -270,22 +270,26 @@ def get_prompt(
              2) the paragraph to revise.
         """
 
-        if env_vars.CUSTOM_PROMPT in os.environ and os.environ[env_vars.CUSTOM_PROMPT].strip() != "":
-            prompt = os.environ[env_vars.CUSTOM_PROMPT]
+        custom_prompt = None
+        if (
+            env_vars.CUSTOM_PROMPT in os.environ
+            and os.environ[env_vars.CUSTOM_PROMPT].strip() != ""
+        ):
+            custom_prompt = os.environ[env_vars.CUSTOM_PROMPT]
```

Comment:
The current logic works, but I think it'd be slightly less verbose like so:

```suggestion
        if ((c := os.environ.get(env_vars.CUSTOM_PROMPT, "").strip()) and c != ""):
            custom_prompt = c
```

(The `:=` is the "walrus operator", introduced in Python 3.8, that both sets a variable and returns its value so it can be used in, say, an `if` expression. `os.environ` is a Dict, so it supports the `.get(<key>, <alt>=None)` method, which returns the second argument if the key given in the first argument isn't found.)

Reply:
Very cool suggestion!